maths/pi_monte_carlo_estimation.py

```diff
@@ -0,0 +1,65 @@
+import random
+
+
+class Point:
+    def __init__(self, x: float, y: float) -> None:
+        self.x = x
+        self.y = y
+
+    def is_in_unit_circle(self) -> bool:
+        """
+        True, if the point lies in the unit circle
+        False, otherwise
+        """
+        return (self.x ** 2 + self.y ** 2) <= 1
```

Comment:
As a side note; `return self.x * self.x + self.y * self.y <= 1` is probably going to be faster and more efficient than using pythons power notation.